### PR TITLE
Update readme file to swanlake alpha4

### PR DESCRIPTION
**Pull request by heshanpadmasiri**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,174 +1,203 @@
...
-3. `make -j`
+2.  `cmake ../ -DCMAKE_BUILD_TYPE=Release -DLLVM_DIR=/usr/local/opt/llvm/lib/cmake/llvm/ -DCMAKE_CXX_STANDARD=14`
+3.  `make -j`
 
 This will build:
-* The Rust runtime : runtime/target/release/libballerina_rt.so
-* The nballerinacc (BIR to LLVM IR converter) app : build/nballerinacc
+
+- The Rust runtime : runtime/target/release/libballerina_rt.so
+- The nballerinacc (BIR to LLVM IR converter) app : build/nballerinacc
```
Comment (ruvi-d) [inline README.md:72]: still a few `*` have been replaced with `-`

File: README.md [hunk 2]
```diff
@@ -1,174 +1,203 @@
...
 ### To codegen the C header for the Rust runtime lib
-1. Install cbindgen
+
+1.  Install cbindgen
 
         cargo install cbindgen
-  * cargo installs the dependencies in user's home directory. Hence, add `~/.cargo/bin` to your `PATH` system variable. `export PATH=$PATH:~/.cargo/bin`
-2. Execute build command
+
+* cargo installs the dependencies in user's home directory. Hence, add `~/.cargo/bin` to your `PATH` system variable. `export PATH=$PATH:~/.cargo/bin`
```
Comment (ruvi-d) [inline README.md:106]: This is no longer a sub bullet point

File: README.md [hunk 3]
```diff
@@ -1,174 +1,203 @@
...
   * `-DSKIP_BIR_GEN` To unset it
 * `-DCLANG_TIDY_OVERRIDE=<compiler name>` : Override default clang-tidy executable name e.g. `-DCLANG_TIDY_OVERRIDE=clang-tidy-11`
-  * `-DCLANG_TIDY_OVERRIDE` To unset it
+  - `-DCLANG_TIDY_OVERRIDE` To unset it
 * `-DCHECK_FILTER="--filter=<regex>"` : Filter tests to run. e.g. `-DCHECK_FILTER="--filter=\"simpleMain.bal\""`, `-DCHECK_FILTER="--filter=\"functions\""`
-  * `-UCHECK_FILTER` To unset it
+  - `-UCHECK_FILTER` To unset it
 * `-DCMAKE_CXX_COMPILER=<compiler name>` : Override default compiler e.g. `-DCMAKE_CXX_COMPILER=clang++-11`, `-DCMAKE_CXX_COMPILER=g++`
-  * `-UCMAKE_CXX_COMPILER` To unset it
+  - `-UCMAKE_CXX_COMPILER` To unset it
```
Comment (ruvi-d) [inline README.md:197]: ditto on `-`

**Review by ruvi-d — CHANGES_REQUESTED** [review]
Suggest that you only fix the relevant lines and not automatically try to fix other formatting issues

File: README.md (hunk 1 above)
Comment (heshanpadmasiri) [reply README.md]: Fixed by reusing the read me from the master

File: README.md (hunk 2 above)
Comment (heshanpadmasiri) [reply README.md]: Fixed by reusing the read me from the master

**Review by ruvi-d — CHANGES_REQUESTED** [review]
Please squash the commits to one.

**Comment by heshanpadmasiri** [comment]
> Please squash the commits to one.

Fixed

**Review by ruvi-d — APPROVED** [review]

**Review by KavinduZoysa — APPROVED** [review]

**Pull request merged**